apps/src/netsim/NetSimConnection.js

```diff
@@ -67,28 +64,17 @@ var NetSimConnection = function (displayName, logger /*=new NetSimLogger(NONE)*/
   this.displayName_ = displayName;
 
   /**
-   * Instance of logging API, gives us choke-point control over log output
-   * @type {NetSimLogger}
-   * @private
-   */
-  this.logger_ = logger;
-  if (undefined === this.logger_) {
-    this.logger_ = new NetSimLogger(console, LogLevel.NONE);
-  }
-
-  /**
-   * Access object for instance lobby
-   * @type {netsimStorage.SharedStorageTable}
+   * Selected instance.
```

Comment:
:/ this keeps coming up, and I suppose I should change the terminology because of it.  I was hoping a reference to [this concept](http://en.wikipedia.org/wiki/Instance_dungeon) would be common enough to map.